cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreImpl.java

```diff
@@ -1423,15 +1423,14 @@ private long insertDocumentHistoryToNewTree(SQLDatabase db, DocumentRevision new
         callable.available = !newRevision.isDeleted();
         long newLeafSeq = callable.call(db);
 
-        pickWinnerOfConflicts(db, docNumericID, newRevision.getId(), previousLeafSeq);
+        pickWinnerOfConflicts(db, docNumericID, newRevision.getId());
```

Comment:
I think we should remove it. looking at the call site of this method, we have already checked if the parent exists, otherwise we wouldn't hit this call. So I think it's safe to remove and it'll save us a database access to :)